koku/reporting/migrations/0264_ocpaws_additional_cost_fields.py

```diff
@@ -0,0 +1,213 @@
+# Generated by Django 3.2.16 on 2022-11-10 13:28
+from django.db import migrations
+from django.db import models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("reporting", "0263_add_cost_categroy"),
+    ]
+
+    operations = [
```

Comment:
I think we want to do this migration using raw sql to add all the fields to the individual table in one go, rather than step-wise like this generated migration. Here is an example of how to write raw-sql migrations:
```
    operations = [
        migrations.RunSQL(
            """
                ALTER TABLE reporting_ocpaws_compute_summary_p
                    ADD COLUMN blended_cost TYPE numeric(33, 15),
                    ADD COLUMN markup_cost_blended TYPE numeric(33, 15),
                    ADD COLUMN savingsplan_effective_cost TYPE numeric(33, 15),
                    ADD COLUMN markup_cost_savingsplan TYPE numeric(33, 15)
            """,
            state_operations=[
                migrations.AddField(
                    model_name="ocpawscomputesummaryp",
                    name="blended_cost",
                    field=models.DecimalField(decimal_places=9, max_digits=24, null=True),
                ),
                migrations.AddField(
                    model_name="ocpawscomputesummaryp",
                    name="markup_cost_blended",
                    field=models.DecimalField(decimal_places=15, max_digits=33, null=True),
                ),
                migrations.AddField(
                    model_name="ocpawscomputesummaryp",
                    name="markup_cost_savingsplan",
                    field=models.DecimalField(decimal_places=15, max_digits=33, null=True),
                ),
                migrations.AddField(
                    model_name="ocpawscomputesummaryp",
                    name="savingsplan_effective_cost",
                    field=models.DecimalField(decimal_places=9, max_digits=24, null=True),
                ),
....
```

Reply:
I actually wasn't familiar with these, I appreciate this feedback @maskarb